core/worker.go

```diff
@@ -733,21 +741,23 @@ func (w *worker) commitTransaction(env *environment, tx *types.Transaction) ([]*
 			env.state.RevertToSnapshot(snap)
 			return nil, err
 		}
+		// check if the to location is valid if an etx is generated in the receipt
```

Comment:
I think this change is unnecessary

Reply:
i agree, once the evm catches, it thats all we need